plugins/scaffolder-backend/package.json

```diff
@@ -30,8 +30,10 @@
   },
   "dependencies": {
     "@backstage/backend-common": "^0.4.3",
+    "@backstage/catalog-client": "^0.3.4",
     "@backstage/catalog-model": "^0.6.1",
     "@backstage/config": "^0.1.2",
+    "@backstage/core": "^0.4.3",
```

Comment:
Nop, this is for frontend packages only 😅

Reply:
I was looking to import `BackstageIdentity`, exported from both `core-api`, that says to use `core` instead and `plugin-auth-backend`. Maybe use `plugin-auth-backend` instead?